Formula/erlang.rb

```diff
@@ -32,6 +33,7 @@ class Erlang < Formula
   depends_on "libtool" => :build
   depends_on "openssl"
   depends_on "fop" => :optional # enables building PDF docs
+  depends_on :java => :optional
```

Comment:
Isn't javac part of the JDK? I named it `with-javac` to just reflect the build flag that erlang provides....